src/schema/controllers/user-controller.js

```diff
@@ -0,0 +1,19 @@
+import db from '../../models'
+
+export default {
+  createUser: function createUser (args) {
+    return db.user.create({
+      firstName: args.firstName,
```

Comment:
Should I make resolver async functions? Or is it better to leave the resolver functions return a Promise? Or it's the same?